elastic-apm-agent/src/test/resources/configuration.asciidoc.ftl

```diff
@@ -123,7 +123,7 @@ Valid options: <#list option.validOptionsLabelMap?values as validOption>`${valid
 # ${option.label}
 #
 </#if>
-# ${option.description?replace("\n", "\n# ", "r")}
+# ${option.description?replace("\n", "\n# ", "r")?replace("`boot_delegation_packages", "boot_delegation_packages")?replace("sun.*, co.elastic.apm.agent.*`", "sun.*, co.elastic.apm.agent.*")}
```

Comment:
That's what I meant with removing the back ticks:

```suggestion
# ${option.description?replace("\n", "\n# ", "r")?replace("`", "")}
```

I didn't test that but it seems it would just remove them everywhere for the properties reference.

Reply:
I understood what you asked and replied:
> because there are tons of them

They make sense in other places that should be left as comments in the properties file. Even if they don't render in some UI to have different font, it is easy to understand they mark areas of code. The difference in this place in the properties file is that it is meant (by me) to be ready for uncommenting.